### PR TITLE
[MAR-3468.8] - team support for admin

### PR DESCRIPTION
**Pull request by Reshma-M-A**
Teams Search will go to a new screen. 
Admin can see all the teams that have been created and completed. 

Briefs will show Team ID and Team Name

### CODE REVIEW

File: app/main/views/buyers.py [hunk 1]
```diff
@@ -263,8 +263,16 @@ def seller_feedback_email(brief_id):
 @role_required('admin')
 def find_team_by_team_id():
     team_id = request.args.get('team_id')
+    if team_id:
+        team_info = data_api_client.req.admin().team(team_id).get()
+        print team_info
```
Comment (malmas18) [inline app/main/views/buyers.py]:
```suggestion
```

File: app/main/views/buyers.py [hunk 2]
```diff
@@ -263,8 +263,16 @@ def seller_feedback_email(brief_id):
 @role_required('admin')
 def find_team_by_team_id():
     team_id = request.args.get('team_id')
+    if team_id:
+        team_info = data_api_client.req.admin().team(team_id).get()
```
Comment (malmas18) [inline app/main/views/buyers.py]: what happens when team_info is null?

File: app/templates/view_team.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    {% call summary.field() %}
+      {{ item.permissions['answer_seller_questions'] }}
+    {% endcall %}
+
+    {% call summary.field() %}
+      {{ item.permissions['create_work_orders'] }}
+    {% endcall %}
+
+    {% endcall %}
+    {% endcall %}
```
Comment (malmas18) [inline app/templates/view_team.html]:
```suggestion
{% endcall %}
```

File: app/main/views/buyers.py (hunk 2 above)
Comment (Reshma-M-A) [reply app/main/views/buyers.py]: fixed

Comment (malmas18) [reply app/main/views/buyers.py]: now an error shows when you just click search....
also it is normally bad practice to have a catch all exception handler.

File: app/main/views/buyers.py [hunk 4]
```diff
@@ -264,7 +264,26 @@ def seller_feedback_email(brief_id):
...
+            )
+
+    except:  # noqa
+        flash('no_team_id', 'error')
+        teams = data_api_client.req.admin().team().get()
+        return render_template(
+            'view_teams.html',
+            teams=teams.get('teams'),
+            team_id=team_id
+        )
```
Comment (malmas18) [inline app/main/views/buyers.py]: i am pretty sure you can simplify the logic here.

File: app/templates/view_team.html [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    Team Name: {{team.name}}
+    <br/> 
+    Team Email Address : {{team.emailAddress}}
+    <br/> 
+    Team Status : {{team.status}}
+  </div>
+
+<div> 
+  <br> 
+  <h3> Team Leads Table</h3>
```
Comment (tobiasbrown) [inline app/templates/view_team.html]:
```suggestion
  <h3>Team Leads Table</h3>
```

File: app/templates/view_team.html [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+    <br/>
+    Team Name: {{team.name}}
+    <br/> 
+    Team Email Address : {{team.emailAddress}}
+    <br/> 
+    Team Status : {{team.status}}
+  </div>
+
+<div> 
+  <br>
```
Comment (tobiasbrown) [inline app/templates/view_team.html]: Does this `<br>` need to be at the start of this `<div>`?

File: app/templates/view_team.html [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+%}
+  {% call summary.row() %}
+    {{ summary.text(item.name) }}
+    {% call summary.field() %}
+      <a href="{{ url_for('.find_user_by_email_address', email_address = item.emailAddress)}}" class="question">{{ item.emailAddress}}</a>    
+    {% endcall %}
+  {% endcall %}
+{% endcall %}
+
+<h3> Team Members' Permissions</h3>
```
Comment (tobiasbrown) [inline app/templates/view_team.html]:
```suggestion
<h3>Team Members' Permissions</h3>
```

File: app/templates/view_team.html [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+    {% call summary.field() %}
+      {{ item.permissions['create_work_orders'] }}
+    {% endcall %}
+
+    {% endcall %}
+{% endcall %}
+</div>
+
+<div>
+  <h3> Team Briefs</h3>
```
Comment (tobiasbrown) [inline app/templates/view_team.html]:
```suggestion
  <h3>Team Briefs</h3>
```

File: app/templates/view_teams.html [hunk 9]
```diff
@@ -20,94 +20,47 @@
...
-    {% endcall %}
-
-    {% call summary.field() %}
-      {{ item.permissions['create_work_orders'] }}
-    {% endcall %}
+  {% with messages = get_flashed_messages(with_categories=true) %}
+  {% if messages %}
+      {% for category, message in messages %}
+          {% if message == 'no_team_id' %}
+              {% set displayed_message = "There are no teams that exist with that ID {}".format(team_id) %}
```
Comment (tobiasbrown) [inline app/templates/view_teams.html]:
```suggestion
              {% set displayed_message = "There are no teams that exist with ID {}".format(team_id) %}
```

File: app/templates/view_teams.html [hunk 10]
```diff
@@ -20,94 +20,47 @@
...
+                {% include "toolkit/notification-banner.html" %}
+   </div>
+          {% endwith %}
+      {% endfor %}
+    {% endif %}
+  {% endwith %}
 
+  <div> 
+    <br> 
+    <h3> List of Teams</h3>
```
Comment (tobiasbrown) [inline app/templates/view_teams.html]:
```suggestion
    <h3>List of Teams</h3>
```

File: app/templates/view_teams.html [hunk 11]
```diff
@@ -20,94 +20,47 @@
...
+      field_headings_visible=True)
+  %}
+    {% call summary.row() %}
+      {{ summary.text(item.agency_name) }}
+      {% call summary.field() %}
+        <a href="{{ url_for('.find_team_by_team_id', team_id = item.team_id) }}" class="question">{{ item.team_id }}</a> 
+      {% endcall %}
+      {{ summary.text(item.team_name) }}
+      {{ summary.text(item.status) }}
+      {% endcall %}
```
Comment (tobiasbrown) [inline app/templates/view_teams.html]: Does it make sense to indent here so that the `endcall` blocks are lined up?

**Pull request merged**